six-get-delegator-id/src/six-get-delegator-id.js

```diff
@@ -25,7 +25,7 @@ pub fun main(account: Address): UInt32 {
 
 class UndefinedConfigurationError extends Error {
     constructor(address) {
-      const msg = `Stored Interaction Error: Missing configuration for ${address}. Please see the following to learn more: https://github.com/onflow/flow-js-sdk/blob/master/six/six-get-delegator-id/README.md`.trim()
+      const msg = `Stored Interaction Error: Missing configuration for ${address}. Please see the following to learn more: https://github.com/onflow/fcl-six/blob/main/six-get-delegator-id/README.md`.trim()
```

Comment:
is branch moving from master to `main`? just want to make sure

Reply:
yes, the `fcl-six` repository already moved from master to main
also previously the interactions were a part of old fcl repository, but now it's a new repo